### ISSUE TITLE
Baldur's Gate 3: load order sorting

### ISSUE THREAD

**Issue by Freghette** [Nexus-Mods/Vortex#16886]
Please make load order sorting based on deployment order. Then deployment rules will affect load order, and this will eliminate the need to export load order before purge operation.

**Comment by ChemGuy1611** [comment]
Deployment order in Vortex is used mainly for resolving conflicting files. It is not the same thing as the file-based Load Order (FBLO) for the game. 

These items are not related, and I don't think the Vortex developers will ever implement something like what you are asking for. It would not be desireable to do so anyway. You can already set up the Load Order using the load order tab. 

Tying the FBLO to the mod deployment order would put a huge burden on the user to manually edit the deployment order for every single mod, rather than using a simple Load Order interface.

**Comment by IDCs** [comment]
Hi there - this feature has been requested for different games over the years, we tend to add a "sort by deploy order" button to the LO action bar for those games, but this adds a new layer of complexity which most users avoid.

To take advantage of this functionality, you'd have to define dependencies in the mods page so that the deploy order matches the envisioned load order; sorting then must be done on the load order page via the added button + final tweaks to the LO.

I think the real question here is why you have to export the load order before a purge operation. If it's because the LO keeps resetting, that's down to a bug in 1.13.3; we have a fix for that in 1.13.4 beta which will be released to everyone in the new year.

But if you don't want to wait that long: https://github.com/Nexus-Mods/Vortex-Staging/releases/download/v1.13.4-beta.1/vortex-setup-1.13.4-beta.1.exe

**Comment by Freghette** [comment]
> I think the real question here is why you have to export the load order before a purge operation. If it's because the LO keeps resetting, that's down to a bug in 1.13.3; we have a fix for that in 1.13.4 beta which will be released to everyone in the new year.

Yes, the LO is resetting. But besides that, there are Nexus Collections. Collection users have to manually download a LO and import it. Or if you just have a lot of mods, sorting with a functionality similar to deployment rules would be much better than manually, don't you agree?

**Comment by Freghette** [comment]
> Deployment order in Vortex is used mainly for resolving conflicting files. It is serial, except when creating dependency rules for conflicting mods. It is not the same thing as the file-based Load Order (FBLO) for the game. You can set up the Load Order using the load order tab.
> 
> Tying the FBLO to the mod deployment order would put a huge burden on the user to manually edit the deployment order for every single mod, rather than using a simple Load Order interface.

I mentioned deployment rules because they are already there and working. Why not to give users the ability to quickly set the position of a record in the LO to the value in the "Deployment order" column on the "Mods" page?

> 
> Plus, it is not easily possible to edit the deployment order for mods that do not have conflicts.

In my experience, Vortex always assigns the correct deployment number (as I currently see in the "Deployment order" column), even if there are no actual file conflicts.

**Comment by IDCs** [comment]
Ha! this line here sold it for me.
> UPD. And if the user has multiple collections, then what? Merge multiple load orders manually via a text editor?

I'll add the ability to sort by deployment order via the action bar on the load order page. This will be added to all games that use FBLO
